easybuild/easyconfigs/e/enable/enable-20160817_kiva.patch

```diff
@@ -0,0 +1,29 @@
+#non-git version (without .git directory) build fails
+#bundled kiva version is tied to enable
```

Comment:
@boegel Yes a `kiva` is icluded as well as `agg` for `kiva`. As far as I know, `Enthought` does not distribute a standalone `kiva`, at the homepage of this package referred to have two parts, `enable` and `kiva`  (see http://code.enthought.com/projects/enable/)